Client/Frontend/Browser/BrowserLongPressHandler.swift

```diff
@@ -0,0 +1,152 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import WebKit
+import UIKit
+
+enum ElementType {
+
+    case ImageSrc(url:NSURL,location:CGRect)
+    case hrefLink(url:NSURL,location:CGRect)
```

Comment:
Nit: Please add spaces after colons and commas (url: NSURL, location: CGRect). Here and throughout.